internal/validate/validate.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 NSONE, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validate
+
+import (
+	"fmt"
+
+	"github.com/ns1/pulsar-routemap/pkg/lg"
+	"github.com/ns1/pulsar-routemap/pkg/validator"
+)
+
+func RunValidateCommand(opts *Options) error {
```

Comment:
Small thing:  I think you can probably just drop this in the existing `cmd.go` file in this package.

Reply:
ah! true
Fixed